src/c++/vdo/user/vdoStatsWriter.c

```diff
@@ -0,0 +1,1447 @@
+// SPDX-License-Identifier: GPL-2.0-only
+/*
+ * Copyright 2023 Red Hat
+ *
+ * ATTENTION!!! ATTENTION!!! ATTENTION!!! ATTENTION!!! ATTENTION!!!
+ *
+ * If you add new statistics, be sure to update the following files:
+ *
+ * ../../../c++/vdo/base/statistics.h
+ * ../../../c++/vdo/base/message-stats.c
+ * ../../../c++/vdo/base/pool-sysfs-stats.c
+ * ../../../c++/vdo/user/messageStatsReader.c
+ * ../../../perl/Permabit/Statistics/Definitions.pm
+ *
```

Comment:
Same ccoment as the messageStatsReader.c, just noting this comment needs to change, too.